module-2/backend/week_two.md

```diff
@@ -1,45 +1,130 @@
 ## Week Two - Module 2 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!). 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON - YOU are a web developer!!!).
 
 Note: When you're done, submit a PR.
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. Assume you have the following model:
+Active Record is an ORM 'system-?', which allows you to connect databases to control systems to maximize efficiency when interacting with a CRUD system - creating, reading, destroying, or updating information in that database
 
+2. Assume you have the following model:
 ```ruby
 class Team << ActiveRecord::Base
+  def self.highest_number_of_players
+  end
+  def self.highest_number_of_players_team_name
+  end
+  def self.highest_score_in_season
+  end
+  def self.highest_score_in_season_team_name
+  end
+  def self.most_common_jersey_color
+  end
+  def self.least_common_jersey_color
+  end
 end
 ```
-
 What are some methods you can call on `Team`? If these methods aren't defined in the class, how do you have access to them?
```

Comment:
looking for more activerecord methods like `all` `find` `where` because we are inheriting from ActiveRecord, we have access to all these methods as well as the class methods (like the ones you created)